import_export_extensions/admin/mixins/base.py

```diff
@@ -35,3 +63,131 @@ def app_label(self):
     def app_model_name(self) -> str:
         """Return url name."""
         return f"{self.app_label}_{self.name}"
+
+
+class BaseImportExportMixin:
+    """Provide common methods for export/import mixin."""
+
+    resource_class: ResourceType = None     # type: ignore
+    resource_classes: typing.Sequence[ResourceType] = []
+
+    def check_resource_classes(
+        self, resource_classes: typing.Sequence[ResourceType],
+    ):
+        """Ensure `resource_classes` is subscriptable."""
+        if resource_classes and not hasattr(resource_classes, "__getitem__"):
+            raise Exception(
+                "The resource_classes field type must be "
+                "subscriptable (list, tuple, ...)",
+            )
+
+    def get_resource_classes(self) -> typing.Sequence[ResourceType]:
+        """Return resource classes."""
+        if self.resource_classes and self.resource_class:
+            raise Exception(
+                "Only one of 'resource_class' and 'resource_classes' "
+                "can be set.",
+            )
+        if hasattr(self, "get_resource_class"):
+            warnings.warn(
+                "The 'get_resource_class()' method has been deprecated. "
+                "Please implement the new 'get_resource_classes()' method",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+            return [self.get_resource_class()]
+        if self.resource_class:
+            warnings.warn(
+                "The 'resource_class' field has been deprecated. "
+                "Please implement the new 'resource_classes' field",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+        if not self.resource_classes and not self.resource_class:
+            return [
+                modelresource_factory(
+                    self.model,
+                    resource_class=resources.CeleryModelResource,
+                ),
+            ]
+        if self.resource_classes:
+            return self.resource_classes
+        return [self.resource_class]
+
+    def get_resource_kwargs(
+        self, request, *args, **kwargs,
```

Comment:
Put each argument on the new line
```suggestion
        self,
        request,
        *args,
        **kwargs,
```